Source/Core/DolphinQt/GameList/GameListModel.cpp

```diff
@@ -187,6 +188,22 @@ QVariant GameListModel::data(const QModelIndex& index, int role) const
       return compression.isEmpty() ? tr("No Compression") : compression;
     }
     break;
+  case Column::TimePlayed:
+    if (role == Qt::DisplayRole || role == SORT_ROLE)
+    {
+      TimePlayed timer(game.GetGameID());
```

Comment:
This feels like an IO hog, you're re-reading and re-parsing the entire `TimePlayed.ini` every time Qt renders this table field. Maybe think about if there's a better way to handle this.